lib/retest/runner.rb

```diff
@@ -12,15 +12,22 @@ class Runner
     def initialize(command, stdout: $stdout)
       @stdout  = stdout
       @command = command
+      if command.hardcoded?
+        self.last_command = command.to_s
+      end
     end
 
     def run_last_command
+      unless last_command
+        return log('Error - Not enough information to trigger a run')
```

Comment:
Change to `Error - Not enough information to run a command. Please trigger a run first.` maybe